### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by nickmask**
- Added Redux
- Added react-redux
- Added redux-logger
- Added a reducer
- Connected a component & tested
- Renamed object to collection item

This closes issue #6

### CODE REVIEW

File: src/views/AppLayout.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ const AppLayout = (props) => {
...
+        {props.children}
       <div>Bottom menu</div>
     </div>
   )
 }
 
-AppLayout.propTypes = {
-  children: React.PropTypes.element.isRequired,
-}
+// AppLayout.propTypes = {
```
Comment (ro-savage) [inline src/views/AppLayout.js:13]: Can be deleted

File: src/views/collectionItems/collectionItem/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import CollectionItemRoute from './CollectionItemRoute'
+import CollectionItemHeader from './CollectionItemHeader/CollectionItemHeader'
+import CollectionItemStats from './CollectionItemStats/CollectionItemStats'
+import CollectionItemAdd from './CollectionItemAdd/CollectionItemAdd'
+import CollectionItemContainer, { CollectionItem } from './CollectionItemContainer/CollectionItemContainer'
+
+
+export { CollectionItemRoute, CollectionItemStats, CollectionItemHeader, CollectionItemAdd, CollectionItem, CollectionItemContainer }
```
Comment (ro-savage) [inline src/views/collectionItems/collectionItem/index.js:8]: Lets put each import on its own line

File: src/views/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    collapsed: true,
+  })
+
+  const store = createStore(
+    rootReducer,
+    applyMiddleware(
+      loggerMiddleware
+    )
+  )
+  console.log(store.getState())
```
Comment (ro-savage) [inline src/views/configureStore.js:17]: This can be removed right?

**Comment by ro-savage** [comment]
Couple minor changes. Then good to merge. Nice work. Approved.

**Comment by nickmask** [comment]
Closed by PR #16

**Pull request merged**